packages/aragon-cli/src/commands/dao_cmds/token_cmds/new.js

```diff
@@ -19,23 +21,28 @@ exports.builder = yargs => {
     .positional('symbol', {
       description: 'Symbol of the new Token',
     })
+    .option('decimal-units', {
+      description: 'Total decimal units the new token will use',
+      default: 18,
+    })
     .option('transfer-enabled', {
       description: 'Whether the new token will have transfers enabled',
+      boolean: true,
       default: true,
     })
-    .option('decimal-units', {
-      description: 'Total decimal units the new token will use',
-      default: 18,
+    .option('token-factory-address', {
+      description: 'Address of the MiniMeTokenFactory',
+      type: 'string',
```

Comment:
Given that we know at least one of these factories exists in both rinkeby and mainnet, it would be nice to default to them if the user's environment matches those networks.

Reply:
I searched on the deployments repo and couldn't found these factories. Are A1 using some deployed by Giveth or someone else?

Reply:
For mainnet, this is the `MiniMeTokenFactory` that all the kits use: https://etherscan.io/address/0x909d05f384d0663ed4be59863815ab43b4f347ec (it's linked in the [deployment documentation](https://github.com/aragon/deployments/tree/master/environments/mainnet#dao-kits-1) for mainnet)

For rinkeby, we don't use a canonical factory as the kit deployment script deploys a new one every time. This is the `MiniMeTokenFactory` that the latest version of `democracy-kit` uses: https://rinkeby.etherscan.io/address/0x6ffeb4038f7f077c4d20eaf1706980caec31e2bf

Reply:
Actually, it may make sense to deploy a new instance of these factories to both rinkeby and mainnet, so we can see how users interact with this command (at least with the default factory specified).